src/reducers/svgSizeReducer/index.js

```diff
@@ -0,0 +1,21 @@
+import * as types from '../../actions/types.js';
+const initState ={
+
+	size: {
+		width:'',
+		height:''
+	}
+
+}
+export function svgSizeReducer(state = initState, action) {
+
+	switch(action.type) {
+		
+		case types.SVG_SIZE:
+		return Object.assign({}, state, { size: action.payload});
```

Comment:
Why we are not using spread operator instead of Object.assign?

Reply:
Okay